include/wifi/KikurageWiFi.h

```diff
@@ -0,0 +1,18 @@
+#ifndef _KikurageWiFi_H_
+#define _KikurageWiFi_H_
+
+#include <M5Stack.h>
+#include <ArduinoJson.h>
+
+struct KikurageWiFi {
+    int totalCount;
+    int count;
+    String ssid;
+    int32_t channel;
+    int32_t rssi;
+    bool isAuthOpen;
+};
+
+String getKikurageWiFiJSONString(KikurageWiFi kikruageWiFi);
+
+#endif // _KkikurageWiFi_H_
```

Comment:
spell miss in line 18.

Reply:
done
https://github.com/shusuke0812/Kikurage-Device/pull/10/commits/84a4154e0299b7c2d4d963f85f139424d50d372f